blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
It seems like you are trying to import 'debug_toolbar_urls' from 'debug_toolbar.toolbar', but this module does not exist. If you want to include Django Debug Toolbar URLs, you should import it from 'debug_toolbar' like this: 'from debug_toolbar import urls as debug_toolbar_urls'.